pkgs/tools/security/trueseeing/default.nix

```diff
@@ -15,22 +15,20 @@ python3.pkgs.buildPythonApplication rec {
     hash = "sha256-g5OqdnPtGGV4wBwPRAjH3lweguwlfVcgpNLlq54OHKA=";
   };
 
-  postPatch = ''
-    substituteInPlace pyproject.toml \
-      --replace "attrs~=21.4" "attrs>=21.4"
-  '';
-
   nativeBuildInputs = with python3.pkgs; [
     flit-core
+    pythonRelaxDepsHook
```

Comment:
Why was this added ? I believe it can be used when we want to be lax on runtime dependencies ?